tests/test_filter_detections_stage.py

```diff
@@ -112,32 +113,26 @@ def test_filter_copy(config):
 @pytest.mark.parametrize('do_copy', [True, False])
 @pytest.mark.parametrize('threshold', [0.1, 0.5, 0.8])
 @pytest.mark.parametrize('field_name', ['v1', 'v2', 'v3', 'v4'])
-def test_filter_column(config, do_copy, threshold, field_name):
-    input_file = os.path.join(TEST_DIRS.tests_data_dir, "filter_probs.csv")
-    df = read_file_to_df(input_file, file_type=FileTypes.Auto, df_type='cudf')
-
+def test_filter_column(config, df, do_copy, threshold, field_name):
     fds = FilterDetectionsStage(config,
                                 threshold=threshold,
                                 copy=do_copy,
                                 filter_source=FilterSource.DATAFRAME,
                                 field_name=field_name)
+    expected_df = df.to_pandas()
```

Comment:
Use marks instead of converting

Reply:
For this test I need the input to be cudf, but I'm using pandas to build out the expected results.